src/backend/access/rmgrdesc/xactdesc.c

```diff
@@ -208,7 +208,9 @@ xact_desc_assignment(StringInfo buf, xl_xact_assignment *xlrec)
 
 static void
 xact_desc_prepare(StringInfo buf, XLogRecord *record) {
+#ifdef USE_ASSERT_CHECKING
 	uint8		info = record->xl_info & ~XLR_INFO_MASK;
```

Comment:
We can tack the unused attribute on, no?

```suggestion
	uint8		info PG_USED_FOR_ASSERTS_ONLY = record->xl_info & ~XLR_INFO_MASK;
```

Reply:
We absolutely can, the only reason I didn't is that I didn't actually know about that one until now =) Will fix that before pushing, thanks for teaching me new tricks!